apps/server/src/shared/domain/types/learnroom.types.ts

```diff
@@ -0,0 +1,11 @@
+export enum LearnroomTypes {
+	'Course' = 'course',
+}
+
+export type LearnroomMetadata = {
+	id: string;
+	type: LearnroomTypes;
+	title: string;
+	shortTitle: string;
+	displayColor: string;
```

Comment:
we also would add the untilDate here right?

Reply:
yes!